svd/extract.sh

```diff
@@ -10,3 +10,4 @@ cp stm32h753x.svd stm32h753v.svd
 cp stm32h7x5_cm4.svd stm32h747cm4.svd
 cp stm32h7x5_cm7.svd stm32h747cm7.svd
 cp stm32h7b3x.svd stm32h7b3.svd
+cp stm32mp157x.svd stm32mp157.svd
```

Comment:
There's no need to copy this, you can just put `stm32mp157x.svd` in the device file. If the 151/153 end up using the same SVD, then it would be appropriate to copy the file to a 151.svd and 153.svd.